Dev mitzi vazquez Entrega Final

mitzyz opened the pull request.

Commit:
Se agrega un enumerador con los valores de las categorías de las películas
Se agrega una función en el enumerador para recibir el indice del segmento y retornar PathMovies
Se agrega seccion de visto reciente

Commit:
Se agregan comentarios
Se agrega apartado para mostrar las películas recientes
Añadir la notificacion para mostrar las películas recientes
se pinta una tabla con las películas recientes

Commit:
se agrega badge y alerta de que se agrego recientemente una película en visto recientemente con su contador

Commit:
Se modifica la función para que el badge no tenga color cuando esta en 0
modificacion del parametro
y se agrega comentario de la función del createBadge

JosueQuinonesRivera commented on BAZProject/Controller/RecentlySeenViewController.swift:
```diff
@@ -0,0 +1,82 @@
+            }
+        }
+    }
+    var countMovies: Int = 0
```
cual es el proposito de este contador?

Commit:
Mover la logica para el badge del tabBarController a la clase TrendingViewController

Commit:
Se agrega clase para el modelo de las películas recientes
Se agrega un protocolo  RecentSeenDataSource con 2 funciones para obtener películas recientes mediante la el modelo de la clase  y para el reseteo del badge
Se agregan 2 función en la clase RecentlySeenTableViewController para seteo del delegado y resetear el badge
Se agregan comentarios
